### ISSUE TITLE
Support custom .exe names

### ISSUE THREAD

**Issue by Pluckerpluck** [Maselkov/GW2RPC#6]
So I happen to have a super weird issue with GW2 where the game will **immediately** crash if I run "gw2-64.exe"

What's extra weird is how it will start working again if I simply rename the exe file, in my case to "gw2-64-working.exe"

The exe file is not changed, just renamed....

----------

To support my super weird issue, is there any chance you could add a "custom" exe name into the config file? 

It would save me having to actually having to build it myself or actually working out why this weird issue exists...

**Comment by Maselkov** [comment]
Check antivirus first I'd say

**Comment by Pluckerpluck** [comment]
I don't use antivirus on this PC. Well other than windows defender which hasn't detected anything.

Quick question: How do you use pyinstaller to actually make the exe? I'm happy to build this just for myself. I'd just rather not have keep powershell open while running it which is the only way I know how currently.

**Comment by Maselkov** [comment]
Point pyinstaller to run.py and it will make the .exe in dist folder, provided you have the correct libraries (which i assume you do since youre running it via powershell)

To get .exe icons you need to do a few more things, but dunno if you need them

**Comment by Pluckerpluck** [comment]
Awesome. Got it working, though it took me a while to work out I needed to specify `pkg_resources` as a hidden import to make it work. Really didn't know much about `pyinstaller` so it was hard to know what to look for.

Got the icon working as well which is nice. After all the effort I went through to get the damn thing to build I thought I might as well work out how to bundle the icon as well.

I'm surprised you haven't added a portable `.spec` file to make building easier. The only thing that doesn't appear portable is the `pathex`, which could be dealt (according to StackOverflow) with something like:

```
import os
spec_root = os.path.abspath(SPECPATH)

...

pathex=[spec_root]
```

Anyway, ty for at least confirming what I was meant to by running pyinstaller against. When it wasn't working it was **really** nice to know that I was at least attempting the right thing.
